Source/WebCore/editing/DeleteSelectionCommand.cpp

```diff
@@ -741,7 +741,7 @@ void DeleteSelectionCommand::handleGeneralDelete()
                             offset = n->computeNodeIndex() + 1;
                     }
                     removeChildrenInRange(*m_downstreamEnd.deprecatedNode(), offset, m_downstreamEnd.deprecatedEditingOffset());
-                    m_downstreamEnd = makeDeprecatedLegacyPosition(m_downstreamEnd.deprecatedNode(), offset);
+                    m_downstreamEnd = makeDeprecatedLegacyPosition(RefPtr { m_downstreamEnd.deprecatedNode() }.get(), offset);
```

Comment:
Seems like `makeDeprecatedLegacyPosition` takes ownership so it should take a `RefPtr` or `RefPtr&&`?